-

```diff
@@ -0,0 +1 @@
="110.400002" class="node-body" x="2579.200195" y="-1809.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="110.400002" class="node-contents" x="2579.200195" y="-1997.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-fake-node="" data-token="asm.label" data-location-references="/function/0x400818:Code_aarch64">root_12_</div></body></foreignObject><rect width="110.400002" class="node-body" x="2579.200195" y="-1997.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="207.600006" class="node-contents" x="1994.399902" y="-2091.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-fake-node="" data-token="asm.label" data-location-references="/function/0x4005fc:Code_aarch64">problem_function_</div></body></foreignObject><rect width="207.600006" class="node-body" x="1994.399902" y="-2091.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="207.600006" class="node-contents" x="1994.399902" y="-2279.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-fake-node="" data-token="asm.label" data-location-references="/function/0x4005fc:Code_aarch64">problem_function_</div></body></foreignObject><rect width="207.600006" class="node-body" x="1994.399902" y="-2279.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="207.600006" class="node-contents" x="1994.399902" y="-2185.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-fake-node="" data-token="asm.label" data-location-references="/function/0x4005fc:Code_aarch64">problem_function_</div></body></foreignObject><rect width="207.600006" class="node-body" x="1994.399902" y="-2185.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="196.800003" class="node-contents" x="1994.400024" y="-4347.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-fake-node="" data-token="asm.label" data-location-references="/function/0x400e38:Code_aarch64">static_init_tls_</div></body></foreignObject><rect width="196.800003" class="node-body" x="1994.400024" y="-4347.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="164.400009" class="node-contents" x="3493.200195" y="-4065.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-token="asm.label" data-location-references="/function/0x400154:Code_aarch64">prefix__start</div></body></foreignObject><rect width="164.400009" class="node-body" x="3493.200195" y="-4065.250000" rx="5" ry="5" height="46.500000"></rect><foreignObject width="196.800003" class="node-contents" x="3493.200195" y="-4159.250000" height="46.500000"><body xmlns=""http://www.w3.org/1999/xhtml""><div data-token="asm.label" data-location-references="/function/0x400170:Code_aarch64">prefix__start_c_</div></body></foreignObject><rect width="196.800003" class="node-body" x="3493.200195" y="-4159.250000" rx="5" ry="5" height="46.500000"></rect></svg>
```

Comment:
Ehm, what is this file?

Reply:
Whoops, I git-added it by mistake it seems, my bad.

Reply:
I suspected that. Feel free to mark this conversation as resolved when you have removed it.

Reply:
Yes-yes, of course, I kept it often as a remainder so that I don't forget 😅

Reply:
It's gone :)